### PR TITLE
filled in TODOs for bugs, emails and data review

### PR DESCRIPTION
**Pull request by betling**
second attempt at updating the yaml files.

### CODE REVIEW

**Comment by betling** [comment]
adding @rhelmer to list of possible reviewers since @knowtheory is probably too busy.  let me know if this is not part of the usual workflow.

**Review by than-mozilla — APPROVED** [review]
Looks good to me.

**Comment by than-mozilla** [comment]
@betling i took a quick look at [the previous commit](https://github.com/mozilla-rally/rally/commit/9fd0b2aeb82ca37f817dcda51bd2f34b6925b487) and where things went awry with [the YAML anchors](https://support.atlassian.com/bitbucket-cloud/docs/yaml-anchors/).

It looks like the commit had a mix of list syntax (the `-`) and alias references, which would probably bork things up.

Assuming that there's a section that looks like this:

```
bugs: &bugs
    - https://bugzilla.mozilla.org/show_bug.cgi?id=1778196
```

Then i believe the entries should work if they look like

```
bugs: *bugs
```

rather than

```
bugs: 
    - *bugs
```

Basically, my understanding is that the `*bugs` syntax is a reference to all the values specified under the block tagged with `&bugs`

**Comment by betling** [comment]
ah, I see.  so it's not just the text, but everything.  got it.  thanks.
